extras/systemd/heketi.service

```diff
@@ -4,7 +4,7 @@ Description=Heketi Server
 [Service]
 Type=simple
 WorkingDirectory=/var/lib/heketi
-EnvironmentFile=-/etc/heketi/heketi.json
+EnvironmentFile=-/etc/heketi/heketi.env
```

Comment:
Hi Talur.  There is no heketi.env installed.  Should we create one, just remove this line completely?